client/utils/index.tsx

```diff
@@ -123,23 +123,23 @@ const fetcher = (url: string) => fetch(url).then((res) => res.json());
 export { fetcher };
 
 export function claimOpensTimestamp() {
-  return parseInt(process.env.CLAIM_OPENS || "0");
+  return parseInt(useStore.getState().claim.claimOpensTs || "0");
 }
 
 export function claimClosesTimestamp() {
-  return parseInt(process.env.CLAIM_CLOSES || "0");
+  return parseInt(useStore.getState().claim.claimClosesTs || "0");
 }
 
-const now = Math.floor(new Date().getTime() / 1000);
+const now = () => Math.floor(new Date().getTime() / 1000);
 
 export function claimOpenTimestampPassed() {
   if (!claimOpensTimestamp()) return true;
-  return now > claimOpensTimestamp();
+  return now() > claimOpensTimestamp();
```

Comment:
This was needed otherwise a "cached" now time was used.

Reply:
Smart.